packages/workbox-routing/src/Router.ts

```diff
@@ -241,28 +263,32 @@ class Router {
    * @param {Object} options
    * @param {URL} options.url
    * @param {Request} options.request The request to match.
-   * @param {FetchEvent} [options.event] The corresponding event (unless N/A).
+   * @param {Event} [options.event] The corresponding event (unless N/A).
```

Comment:
Should this be `ExtendableEvent` to match the argument type?

Reply:
I could, but elsewhere I've been keeping the JSDoc comments simple by using `Event`. I can't use `Event` in TS because it'll say things like `event.request` doesn't exist, so I have to be specific.